tailwind.config.js

```diff
@@ -11,5 +13,13 @@ module.exports = {
       },
     },
   },
-  plugins: [],
+  plugins: [
+    plugin(function ({ addUtilities, addComponents, e, config }) {
+      // Add your custom styles here
```

Comment:
I don't see you use any plugins. Can you add one?

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/0dd59e9b56006937e1ee0f6441b2373747107da0